recipes-core/mongoose/mongoose_7.10.bb

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: MIT
+# Copyright (C) 2021 iris-GmbH infrared & intelligent sensors
+
+require recipes-core/mongoose/mongoose.inc
+
+SRCREV = "b379816178abdcd59135aa32f990a4b3bbbfb54b"
```

Comment:
Commit ID von welchem branch? Mongoose?

Reply:
Die 7.10 upstream

Reply:
@AnTranExScience see the include file mongoose.inc, which defines the `SRC_URI`